client/src/component/Home/CompanyHome.jsx

```diff
@@ -1,18 +1,89 @@
-import React, { Component, Fragment } from 'react';
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import Moment from "react-moment";
 
-class CompanyDashboard extends Component {
-  state = {
-    
-  }
+import axios from "axios";
+
+import {
+  MainContainer,
+  SmallCard,
+  MappedList,
+  MappedItem
+} from "../Styles/CardContainers";
+import { Title } from "bloomer/lib/elements/Title";
+
+export default function CompanyHome() {
+  const endpoint = "http://localhost:3000";
```

Comment:
just noticed you did this pattern with the endpoint. makes it easy to change if we change this in the future